roster/models.py

```diff
@@ -0,0 +1,71 @@
+from django.db import models
+from uuid import uuid4
+from django.contrib.auth.models import User
+
+class Site(models.Model):
+    name = models.CharField(max_length=100)
+    site_id = models.CharField(unique=True, default=uuid4, max_length=36)
+    # TODO: is this an address field? Should it be managed as such?
+    location = models.TextField(max_length=500)
+
+    # TODO: should these three be restricted by choices?
+    language_code = models.CharField(max_length=2, default='EN')
+    country_code = models.CharField(max_length=2, default='US')
+    timezone = models.CharField(max_length=30, default='EST')
```

Comment:
timezone should use the official "tz" code, eg "America/New_York" rather than "EST".
https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

Reply:
I've now used `pytz` to generate this as a list of choices.